app/dropshop/accounts/authentication.py

```diff
@@ -58,6 +58,8 @@ def forgot_password_user_check(request, email):
                 break
         forgot_password_email = ForgotPasswordEmail(confirmation_code=confirmation_code, user=user)
         forgot_password_email.save()
+        user.is_active = True
+        user.save()
```

Comment:
I thought we were going to add messaging if a user tries to login when they're inactive?

Reply:
When they go to the link for forgot password it will reactive their account

Reply:
But how will they know to do that?
They will try to login first right?

Reply:
If you address that case, we can approve and merge.